app/views/browse/deployment.html

```diff
@@ -75,25 +75,62 @@
           <div ng-if="!loaded">Loading...</div>
           <div class="row" ng-if="loaded">
             <div class="col-md-12" ng-class="{ 'hide-tabs' : !deployment }">
+              <div ng-if="deployment.spec.paused" class="alert alert-info animate-if">
+                <span class="pficon pficon-info" aria-hidden="true"></span>
+                <strong>{{deployment.metadata.name}} is paused.</strong>
+                This will pause any in-progress rollouts and stop new
+                rollouts from running until the deployment is resumed.
+                <span ng-if="!updatingPausedState && ({ group: 'extensions', resource: 'deployments' } | canI : 'update')">
+                  <a href="" ng-click="setPaused(false)" role="button">Resume deployment</a>
+                </span>
+              </div>
               <uib-tabset>
-                <uib-tab active="selectedTab.details">
-                  <uib-tab-heading>Details</uib-tab-heading>
+                <uib-tab active="selectedTab.replicaSets">
+                  <uib-tab-heading>Replica Sets</uib-tab-heading>
```

Comment:
how would you feel about "History" instead.  I don't really like the tab being called Replica Sets